libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -371,6 +372,11 @@ export class CosmosAPI {
 
     if (data.tx_response.code != 0) {
       // error codes: https://github.com/cosmos/cosmos-sdk/blob/master/types/errors/errors.go
+      // Handle cosmos sequence mismatch error(error code 32) because the backend returns a wrong sequence sometimes
+      // This is a temporary fix until we have a better backend
+      if (data.tx_response.code === 32) {
```

Comment:
We could add unit test like it's done here : https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/families/cosmos/api/Cosmos.test.ts


Minor